benchto-driver/src/main/java/com/teradata/benchto/driver/loader/BenchmarkLoader.java

```diff
@@ -272,23 +305,13 @@ private String benchmarkName(Path benchmarkFile)
         return variableMapList;
     }
 
-    private Predicate<Path> activeBenchmarks()
-    {
-        Optional<List<String>> activeBenchmarks = properties.getActiveBenchmarks();
-        if (activeBenchmarks.isPresent()) {
-            return benchmarkNameIn(activeBenchmarks.get());
-        }
-        return path -> true;
-    }
-
-    private Predicate<Path> benchmarkNameIn(List<String> activeBenchmarks)
+    private List<String> activeBenchmarkPatterns()
```

Comment:
Not really. The semantics didn't change, the mechanics did, as the pattern is now evaluated during scanning for files.

Perhaps I could spare doing this at all though, so my change would be smaller.